CONTRIBUTE.md

```diff
@@ -72,15 +72,15 @@ The downloadable Spigot plugins and their meta info are stored on our server, ho
 
 If you want to contribute your network traffic by setting up a mirror, feel free to check out [How to setup a mirror](#how-to-set-up-a-mirror)
 
-### How to set up a mirror
+### How to setup a mirror
 
 The MCPRS server is hosted with a plain file server that supports both http and rsync. The official server is hosted using Nginx, but any file server with such compatibility would work. You can follow one of the approaches below to set up a mirror.
 
 After setting up a mirror, if you want to add it to our mirror list, you can submit a pull request to this repo editing the [mirrorlist.yml](mirrorlist.yml) file.
 
 #### Setup Mirror using Docker Compose
 
-For convenience, we created a docker image so that you can set up a mirror using Docker. It will automatically set up:
+For convenience, we created a docker image so that you can setup a mirror using Docker. It will automatically set up:
```

Comment:
https://www.merriam-webster.com/dictionary/setup
setup is a noun, set up is the verb

Reply:
This was actually marked by the IntelliJ spellcheck :P Do you want me to change it back?

Reply:
IntelliJ has skill issue I guess we can change it later